README.md

```diff
@@ -1,2 +1,14 @@
 # jtu-22-git-docker-assignment
 Git+Docker Assignment for jTU - '22
+
+git checkout develop
+
+- changes to utils.py
+
+git add .
+
+git commit -m "Added my name to str in utlis.py"
+
+docker build -t check .
+
+docker run -it --rm check
```

Comment:
You need to use the `-p` flag for `docker run` to publish ports of your container when running it from a Docker image, in order to have them be accessible from the outside world.
Take a look at this link for more info: https://docs.docker.com/config/containers/container-networking/#published-ports

Reply:
So some remote device can interact with the app running in docker only when port is exposed right?

and if we do not expose it does it run on localhost?